getting_hired/project_portfolio.md

```diff
@@ -14,6 +14,38 @@ Remember to write some documentation for both technical and non-technical audien
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
 
+### Design Inspirations
```

Comment:
A brief sentence or two to introduce what's in the drop-down belongs here.

Reply:
Added an intro line